app/models/survey_response.rb

```diff
@@ -46,7 +46,8 @@ def self.to_csv
         csv.push(row)
       end
     end
-    csv.join("\n")
+    csv.insert(1,'ID of Survey Response in the Database,ID of the Survey associated with the survey response in the database.,Distance from street intersection near the home and the destination school as calculated by the Google Maps API,Point location of the intersection near the survey respondent’s home. Provided on a 4326 projected plane.,When the survey response was created.,When the survey response was last modified.,Integer value that enumerates the type of walk shed the point is in. ,Do you usually drop off your child on your way to work or another destination?,How does your child get home FROM school on most days?,What grade is your child in?,Do you usually pick up your child on your way from work or another origin?,How does your child get TO school on most days?,ID of the school associated with the survey in the database.,How many people in your household have a driver license?,How many vehicles do you have in your household?,DOE Org Code for the Respondent’s School')
+    csv.join
```

Comment:
Why aren't we joining on newline anymore? Will CSV work without it?

Reply:
This method was not attempting to join the CSV "on the newlines" but rather is saying "join the array elements together and insert a new line between them" which leads to output that has an extra newline between each row. Eliminating the argument generates output in the expected manner because the end of each CSV row already has a `\n` at the end of it.